app/views/curation_concerns/file_sets/upload/_form_fields.html.erb

```diff
@@ -2,7 +2,7 @@
       <%= hidden_field_tag(:total_upload_size, 0) %>
       <%= hidden_field_tag(:relative_path) %>
       <%= hidden_field_tag(:upload_set_id, @upload_set_id) %>
-      <%= hidden_field_tag(:parent_id, @work_id) %>
+      <%= hidden_field_tag(:parent_id, @presenter.id) %>
```

Comment:
I don't understand why having the hidden field is necessary. It should be passed in the route:
```
curation_concerns_file_sets POST   /concern/container/:parent_id/file_sets
```